### ISSUE TITLE
DOM event throws 'selectionDirection' property from 'HTMLInputElement'

### ISSUE THREAD

**Issue by kamesh-a** [nwjs/nw.js#2934]
```
Uncaught InvalidStateError: Failed to read the 'selectionDirection' property from 'HTMLInputElement': The input element's type ('hidden') does not support selection
```

Events stops bubbling. So we are unable to capture contextMenu events, In our application we have build routine on top of this event to support contextMenu to popup inside iframes.

Working Build :  nw-0.10.x, nw.0.10.6-pre
Test case Fails : nw-0.11.x, nw.0.11.5

Reproducible App : 
https://dl.dropboxusercontent.com/u/42787852/nw-crash/selectionDirection_prop_missing_error.zip

Thanks.
##

**Comment by ags131** [comment]
Also, Chrome does not throw any exception for the same test case

**Comment by adam-lynch** [comment]
Related: #2846

**Comment by busykai** [comment]
@rogerwang, this issue causes some of our tests to fail. i've looked into this a little and i cannot get my head around this logic ([ConsoleBase.cpp source](https://github.com/nwjs/blink/blob/nw12/Source/core/frame/ConsoleBase.cpp#L221-L224)):

```cpp
    String message;
    bool gotStringMessage = arguments ? arguments->getFirstArgumentAsString(message) : false;
    if (gotStringMessage) {
       message = "";
```
it does seem like a typo to me: if we've gotten the string, why would we want to jsonify it instead? shouldn't the check be `!gotStringMessage`? it is the root cause of this. i assume the jsonification should be attempted if `console.log` has been executed in `node` context in which case `getFirstArgumentAsString` would return `false` since the worlds are different. once you confirm, i can submit a PR to fix it in the versions previous to v0.13 (if needed) and fix it in my branch as well.
